### ISSUE TITLE
Transcoded ffmpeg audio not working in 1.3.1

### ISSUE THREAD

**Issue by Mr-HaleYa** [AlexxIT/go2rtc#309]
I'm using a Reolink E1 Zoom and HassOS on an i5-6500T mini Pc

I was using the WebRTC Camera integration that has go2rtc 1.2.0 and had the go2rtc config set like this 

```
streams:
  lvrm_cam:
    - rtsp://admin:password@192.168.50.164:554/h264Preview_01_main
    - ffmpeg:rtsp://admin:password@192.168.50.164:554/h264Preview_01_main#audio=opus

rtsp:
  listen: ":8554"
  default_query: "video&audio=opus"


webrtc:
  listen: ":8555"
  candidates:
    - stun:8555
```

this works great for getting an RTC connection and audio out of the webRTC Lovelace card

```
type: custom:webrtc-camera
entity: camera.lvrm_cam
poster: null
ui: true
muted: true
mode: webrtc
```

I wanted to try out the go2rtc 1.3.0 to see if it would fix my problems with connecting outside the network and it did. I uninstalled WebRTC Camera and installed go2rtc 1.3.0 then reinstalled WebRTC Camera and it identified the rtsp server of go2rtc when adding the integration and everything worked, I had less than 1-sec latency from 3 states away. Everything except audio... I read through both the go2rtc forum post and the WebRTC camera forum post and messed with it for 11+ hours and no matter what I do I can't get audio in the WebRTC Camera card. 

I can get audio in the `camera.lvrm_cam` stream in the UI but when I add that entity to the card it creates a new stream in the UI `lvrm_cam` and that stream has no audio. 

All streams do have audio in MSE mode just nothing in RTC

Any help would be appreciated

**Comment by AlexxIT** [comment]
I don't understand where you get `camera.lvrm_cam` and why you even need it. You can use:
```
type: custom:webrtc-camera
url: lvrm_cam
```

**Comment by Mr-HaleYa** [comment]
I created a generic camera like shown [here ](https://github.com/AlexxIT/go2rtc#module-hass)

```
1. Camera any source => go2rtc config => Generic Camera
- Install any go2rtc
- Add your stream to go2rtc config
- Hass > Settings > Integrations > Add Integration > Generic Camera > rtsp://127.0.0.1:8554/camera1 (change to your stream name)
```

Yes, doing this does work in 1.2.0 and 1.3.0 for getting audio and video 
```
type: custom:webrtc-camera
url: lvrm_cam
```

but why does the generic camera setup give audio and video in 1.2.0 but not in 1.3.0?

**Comment by Mr-HaleYa** [comment]
ya even in 1.3.1 if you create a generic camera from the rtsp stream created when adding a camera in config the camera won't have sound.


I get this. The bytes count for sending video goes up but the audio remains at 0 
```
{
  "producers": [
    {
      "type": "RTSP active producer",
      "url": "rtsp://127.0.0.1:8554/lvrm_cam",
      "remote_addr": "127.0.0.1:8554",
      "user_agent": "go2rtc/1.3.1",
      "medias": [
        "video, recvonly, H.264 High 5.1",
        "audio, recvonly, OPUS/48000/2"
      ],
      "receivers": [
        "96 H264, bytes=3714859, senders=1",
        "97 OPUS/48000/2, bytes=0, senders=1"
      ],
      "recv": 3818204
    }
  ],
  "consumers": [
    {
      "type": "WebRTC/WebSocket async passive consumer",
      "remote_addr": "udp4 prflx 192.168.50.3:56318 related :0",
      "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36",
      "medias": [
        "video, sendonly, VP8, RTX, VP9, H264, AV1, RED, ULPFEC, FLEXFEC-03",
        "audio, sendonly, OPUS/48000/2, RED/48000/2, G722/8000, PCMU/8000, PCMA/8000, CN/8000, TELEPHONE-EVENT/48000, TELEPHONE-EVENT/8000"
      ],
      "senders": [
        "96 H264, bytes=3712631, receivers=1",
        "97 OPUS/48000/2, bytes=0, receivers=1"
      ],
      "send": 3750602
    }
  ]
}
```

**Comment by NickM-27** [comment]
So one difference seems to be that my cameras are all AAC and I am transcoding to OPUS for webrtc, in that case ffmpeg audio is working just fine.

```
    {
      "type": "WebRTC/WebSocket async passive consumer",
      "remote_addr": "tcp4 prflx 192.168.50.8:55037 related :0",
      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36",
      "medias": [
        "video, sendonly, VP8, RTX, VP9, H264, AV1, RED, ULPFEC, FLEXFEC-03",
        "audio, sendonly, OPUS/48000/2, RED/48000/2, G722/8000, PCMU/8000, PCMA/8000, CN/8000, TELEPHONE-EVENT/48000, TELEPHONE-EVENT/8000"
      ],
      "senders": [
        "96 H264, bytes=3037390, receivers=1",
        "96 OPUS/48000/2, bytes=42712, receivers=1"
      ],
      "send": 3114426
    }
```

Seems the issue may be restricted to cameras that serve other types of audio and are being transcoded to aac

**Comment by Mr-HaleYa** [comment]
maybe that is it. I do not have any cameras that output ACC 
The reolink E1 zoom outputs this by default

```
{
  "producers": [
    {
      "type": "RTSP active producer",
      "url": "rtsp://192.168.50.163/h264Preview_01_main/",
      "remote_addr": "192.168.50.163:554",
      "user_agent": "go2rtc/1.3.1",
      "medias": [
        "video, recvonly, H.264 High 5.1",
        "audio, recvonly, MPEG4-GENERIC/16000"
      ],
      "receivers": [
        "96 H264, bytes=49882980, senders=1",
        "97 MPEG4-GENERIC/16000, bytes=765744, senders=1"
      ],
      "recv": 51096684
    }
  ]
}
```

**Comment by NickM-27** [comment]
> maybe that is it. I do not have any cameras that output ACC 
> The reolink E1 zoom outputs this by default
> 
> ```
> {
>   "producers": [
>     {
>       "type": "RTSP active producer",
>       "url": "rtsp://192.168.50.163/h264Preview_01_main/",
>       "remote_addr": "192.168.50.163:554",
>       "user_agent": "go2rtc/1.3.1",
>       "medias": [
>         "video, recvonly, H.264 High 5.1",
>         "audio, recvonly, MPEG4-GENERIC/16000"
>       ],
>       "receivers": [
>         "96 H264, bytes=49882980, senders=1",
>         "97 MPEG4-GENERIC/16000, bytes=765744, senders=1"
>       ],
>       "recv": 51096684
>     }
>   ]
> }
> ```
> 

That is aac audio there

**Comment by Mr-HaleYa** [comment]
... oh ... well in that case any other ideas for the cause?

**Comment by aftertommy** [comment]
I just came across this after debugging my setup for the past few hours. I have a unifi protect camera which VLC is claiming provides both aac and opus audio streams. I set it up as a stream in my config:

```
    front:
        - rtspx://192.168.0.1:7441/NS2a6YJHuyLiU8wM
```

Accessing the stream via rtsp on vlc seems to work just fine. I can select for aac or opus (like so):

```
rtsp://myurl:8554/front?video&audio=opus
```

I had initially set up a Lovelace card using a generic camera entity (for the same reasons as [@Mr-HaleYa ](https://github.com/AlexxIT/go2rtc/issues/309#issuecomment-1484298989)), which resulted in no audio. Then I tried using combinations of the above rtsp url, which also had no audio.

I didn't realize I could just refer to the stream directly, and that appears to have resolved the issue. I had missed [this](https://github.com/AlexxIT/WebRTC#custom-card) bit in the WebRTC Camera documentation:

```
As a url you can use any protocol supported in go2rtc or specify the stream name from the go2rtc config
```

I'm a new user to go2rtc, so 1.3.1 is my first version. If rtsp links are _supposed_ to work with audio, and if they were working previously, then I'm happy to provide any additional information to help resolve things. 

Forgive me in advance if it's clear that I'm just doing something wrong or expecting behavior that's just not supported for my camera setup.
